is_core/generic_views/form_views.py

```diff
@@ -571,13 +574,38 @@ def get_success_url(self, obj):
         else:
             return self.request.get_full_path()
 
+    def _generate_export_types(self, export_types):
+        generated_export_types = []
+        for export_type in export_types or []:
+            generated_export_type = list(export_type)
+            try:
+                generated_export_type.append(get_converter(export_type[1]).media_type)
+            except KeyError:
+                raise ImproperlyConfigured()
+            generated_export_types.append(generated_export_type)
+        return generated_export_types
+
+    def _get_export_types(self):
+        return self._generate_export_types(self.core.get_detail_export_types())
```

Comment:
return self.export_types if self.export_types is not None else self.core.get_detail_export_types()

Reply:
nechapu, to generate musi probehnout, pridava to tam tu chybejici cast, ta tam nikdy nebude jenom v self.export_types